website/docs/api/themes/theme-configuration.md

```diff
@@ -757,6 +757,32 @@ module.exports = {
 };
 ```
 
+## Table of Contents {#table-of-contents}
+
+You can adjust the table of contents via `themeConfig.tableOfContents`.
+
+<small>
+
+| Name       | Type     | Default | Description                            |
+| ---------- | -------- | ------- | -------------------------------------- |
+| `maxDepth` | `number` | `3`     | Max heading level displayed in the TOC |
```

Comment:
I wonder if this should be named `maxLevel`. The number actually corresponds to the highest heading level. I think the unified AST has this property as `depth`, so I used `maxDepth`, but it might be confusing to users that `maxDepth:2` corresponds to a flat TOC (only H2)
 
Also, how can I document that 2-6 is the good value range? Suggestions appreciated.

Reply:
I think we should use `maxDepth` and `maxDepth: 1` to mean only `h2` nodes🧐

You can just say "Number of heading levels displayed in the TOC. Should be an integer between 1 and 5", there's no format requirement about anything

Reply:
Addressed in 7269c0f

I went for `maxHeadingLevel` to be _super_ explicit

Reply:
Not sure what's best but using `minHeadingLevel` & `depth` (`>= 0`) is also an option.

`maxHeadingLevel` can be computed from `minHeadingLevel + depth`

This feature is not so simple to design IIMHO, there are some edge cases (like the user using h2, then h4). 

The API has potential implications on the rendering logic for such edge-cases